src/Presenter/Store/Modules/ConfigurationModule.php

```diff
@@ -51,8 +57,9 @@ class ConfigurationModule implements PresenterInterface
      * @param ExpressCheckoutConfiguration $ecConfiguration
      * @param PayPalConfiguration $paypalConfiguration
      */
-    public function __construct(ExpressCheckoutConfiguration $ecConfiguration, PayPalConfiguration $paypalConfiguration, FundingSourceProvider $fundingSourceProvider)
+    public function __construct(PayPalPayIn4XConfiguration $payIn4XConfiguration, ExpressCheckoutConfiguration $ecConfiguration, PayPalConfiguration $paypalConfiguration, FundingSourceProvider $fundingSourceProvider)
     {
```

Comment:
Phpdoc should be updated too
```suggestion
    public function __construct(
        PayPalPayIn4XConfiguration $payIn4XConfiguration,
        ExpressCheckoutConfiguration $ecConfiguration,
        PayPalConfiguration $paypalConfiguration,
        FundingSourceProvider $fundingSourceProvider
    ) {
```